src/archanaNJun20/PalindromeString.java

```diff
@@ -0,0 +1,23 @@
+package archanaNJun20;
+
+public class PalindromeString {
+	void palindromeString(String inputString){ 
+		String originalstring=inputString;
+		int length=inputString.length();
+		String newString = "";
+		for(int index=length-1;index>=0;index--){
+			char ch=inputString.charAt(index);
+			newString=newString+String.valueOf(ch);
+		}
+		if(newString.equalsIgnoreCase(originalstring))
```

Comment:
you can directly use like this newString.equalsIgnoreCase(inputString)
instead of creating new variable "originalstring". As you are not changing value of inputString variable.

Reply:
The code is updated based on the comment. Thank you for your Suggestion.

Reply:
equalsIgnoreCase never used in palindrome string. Always use .equals()